satpy/etc/readers/viirs_l2.yaml

```diff
@@ -0,0 +1,126 @@
+  fsnrad_l2_viirs:
+    file_reader: !!python/name:satpy.readers.viirs_l2.VIIRSL2FileHandler
+    file_patterns:
+      - 'FSNRAD_L2_VIIRS_CRIS_SS_{spacecraft_name:s}.A{start_time:%Y%j.%H%M}.{collection:03d}.{production_time:%Y%j%H%M%S}.nc'
+
+datasets:
+  cld_lon:
+    name: cld_lon
+    resolution: 1000
+    file_type: [cldmsk_l2_viirs, cldprop_l2_viirs]
+    file_key: geolocation_data/longitude
+    units: degrees
+    standard_name: longitude
+  cld_lat:
+    name: cld_lat
+    resolution: 1000
+    file_type: [cldmsk_l2_viirs, cldprop_l2_viirs]
+    file_key: geolocation_data/latitude
+    units: degrees
+    standard_name: latitude
+  aerdb_lon:
+    name: aerdb_lon
+    resolution: 1000
+    file_type: [aerdb_l2_viirs, aerdb_l2_viirs_nrt]
+    file_key: Longitude
+    units: degrees
+    standard_name: longitude
+  aerdb_lat:
+    name: aerdb_lat
+    resolution: 1000
+    file_type: [aerdb_l2_viirs, aerdb_l2_viirs_nrt]
+    file_key: Latitude
+    units: degrees
+    standard_name: latitude
+  aerdt_lon:
+    name: aerdt_lon
+    resolution: 1000
+    file_type: [aerdt_l2_viirs]
+    file_key: longitude
+    units: degrees
+    standard_name: longitude
+  aerdt_lat:
+    name: aerdt_lat
+    resolution: 1000
+    file_type: [aerdt_l2_viirs]
+    file_key: latitude
+    units: degrees
+    standard_name: latitude
+
+##################################
+# Datasets in file cldmsk_l2_viirs
+##################################
+  Clear_Sky_Confidence:
+    name: Clear_Sky_Confidence
+    long_name: VIIRS Clear Sky Confidence
+    units: None
+    coordinates: [cld_lon, cld_lat]
+    file_key: geophysical_data/Clear_Sky_Confidence
+    file_type: cldmsk_l2_viirs
+
+
+###################################
+# Datasets in file cldprop_l2_viirs
+###################################
+  Cloud_Top_Height:
+    name: Cloud_Top_Height
+    long_name: Cloud Top Height from NOAA CLAVR-x AWG algorithm
+    units: m
+    coordinates: [cld_lon,cld_lat]
+    file_key: geophysical_data/Cloud_Top_Height
+    file_type: cldprop_l2_viirs
+
+##########################################
+# Datasets in files aerdb_l2_viirs and nrt
+##########################################
+  Angstrom_Exponent_Land_Ocean_Best_Estimate:
+    name: Angstrom_Exponent_Land_Ocean_Best_Estimate
+    long_name: Deep Blue/SOAR Angstrom exponent over land and ocean
+    units: None
+    coordinates: [aerdb_lon,aerdb_lat]
+    file_key: Angstrom_Exponent_Land_Ocean_Best_Estimate
+    file_type: [aerdb_l2_viirs, aerdb_l2_viirs_nrt]
+
+  Aerosol_Optical_Thickness_550_Land_Ocean:
+    name: Aerosol_Optical_Thickness_550_Land_Ocean_Best_Estimate
+    long_name: Deep Blue/SOAR aerosol optical thickness at 550 nm over land and ocean
+    units: None
+    coordinates: [aerdb_lon,aerdb_lat]
+    file_key: Aerosol_Optical_Thickness_550_Land_Ocean_Best_Estimate
+    file_type: [aerdb_l2_viirs, aerdb_l2_viirs_nrt]
```

Comment:
Long_name does match what is in the file. Everything is now "1" instead of None. For the AERDB products it was "1" in the file so that was an oversight on my part. For Clear_Sky_Confidence it is None in the file but "1" should be good for the CF compatibility you mentioned.

Reply:
In general I think my preference is for the YAML to only specify metadata that we know needs to be overwritten. For example when "units" are missing or are incorrect (not CF compliant, typos, etc). When you say the units for `Clear_Sky_Confidence` is None, do you mean the string "None", "none", it doesn't exist, or something else?

Reply:
In the file the units for Clear Sky Confidence are equal to the string "none". I'm not really sure of the distinction or best practices of what to use between the different nones.

Reply:
I think CF-wise the unit "1" literally means unitless which I think (at least in English) is equivalent to what is intended with "none".